mono/utils/mono-time.c

```diff
@@ -142,7 +142,16 @@ gint64
 mono_100ns_ticks (void)
 {
 	struct timeval tv;
-#ifdef CLOCK_MONOTONIC
+#if defined(PLATFORM_MACOSX)
+	/* http://developer.apple.com/library/mac/#qa/qa1398/_index.html */
+	static mach_timebase_info_data_t timebase;
+	guint64 now = mach_absolute_time ();
```

Comment:
Hmm, I'm not too familiar with how this method is used but according to the man pages CLOCK_MONOTONIC behaves the same so at least those two codepaths are consistent. This doesn't apply to gettimeofday() of course but not sure on which platforms we fallback to that.